chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+There are two main cases when the raw data will be modified during data cleaning.
+The first one is when there are duplicated entries in the data.
+Ensuring that observations are uniquely and fully identified\sidenote{\url{https://dimewiki.worldbank.org/wiki/ID\_Variable\_Properties}}
+is possibly the most important step in data cleaning
+(as anyone who ever tried to merge data sets that are not uniquely identified knows).
+Modern survey tools create unique observation identifiers.
+That, however, is not the same as having a unique ID variable for each individual in the sample.
+You want to make sure the data set has a unique ID variable
+that can be cross-referenced with other records, such as the Master Data Set\sidenote{\url{https://dimewiki.worldbank.org/wiki/Master\_Data\_Set}}
+and other rounds of data collection.
+\texttt{ieduplicates} and \texttt{iecompdup}, 
+two Stata commands included in the \texttt{iefieldkit} 
+package,\index{iefieldkit}
+create an automated workflow to identify, correct and document
+occurrences of duplicate entries. 
 
-Once the de-identified dataset is created, data must be \textbf{cleaned}.\sidenote{
-\url{https://dimewiki.worldbank.org/wiki/Data_Cleaning}}
-\index{data cleaning}
-This process does not involve the creation of any new data fields.
-Cleaning can be a long process: this is the phase at which
-you obtain an extensive understanding of the contents and structure
-of the data you collected in the field.
-You should use this time to understand the types of responses collected,
-both within each survey question and across respondents.
-Understanding this structure will make it possible to do analysis.
+Looking for duplicated entries is usually part of data quality monitoring,
+as is the only other reason to change the raw data during cleaning:
+correcting mistakes in data entry.
+During data quality monitoring, you will inevitably encounter data entry mistakes,
+such as typos and inconsistent values.
+If you don't, you are probably not doing a very good job at looking for them.
+These mistakes should be fixed in the cleaned data set,
+and you should keep a careful record of how they were identified,
+and how the correct value was obtained.
 
-There are a number of tasks involved in cleaning data.
+% Data description ------------------------------------------------------------------
+Note that if you are using secondary data, 
+the tasks described above can likely be skipped.
+However, the last step of data cleaning, describing the data,
+will probably still be necessary.
+This is a key step to making the data easy to use, but it can be quite repetitive.
 The \texttt{iecodebook} command suite, part of \texttt{iefieldkit},
 is designed to make some of the most tedious components of this process,
 such as renaming, relabeling, and value labeling,
 much easier (including in data appending).\sidenote{\url{https://dimewiki.worldbank.org/wiki/iecodebook}}
```

Comment:
_(including in data appending)_ - I know this is not incorrect, but it doesn't really fit the flow of the sentence. I think that this should be made it's own sentence and use a few more words saying that the process can be done slightly differently for different data sets, so that they can be appended. Or something like that.

Reply:
Data appending is actually of construction, so I just removed the sentence in c80a095.